### ISSUE TITLE
height doesn't work

### ISSUE THREAD

**Issue by foray1010** [tzvetkoff/im2a#3]
the following command doesn't work, it won't output with 40 row, same for width
```
im2a --height=40 --grayscale https://cdn.meme.am/cache/instances/folder668/500x/63425668.jpg
```

**Comment by tzvetkoff** [comment]
Well, yes - currently `--width` & `--height` only work together (and only in terminal output, not in HTML.)
I'll push a change (and tag 0.4.3) later today with a fix for that.

**Comment by tzvetkoff** [comment]
OK, it's fixed in `master` / `0.4.3`.

**Comment by foray1010** [comment]
work perfectly, thanks!
